privacyidea/static/components/config/controllers/ldapMachineResolverController.js

```diff
@@ -38,6 +38,7 @@ myApp.controller("ldapMachineResolverController", function ($scope,
             $scope.params = resolver.data;
             $scope.params.type = 'ldap';
             $scope.params.NOREFERRALS = ($scope.params.NOREFERRALS == "1");
+            $scope.params.EDITABLE = ($scope.params.EDITABLE == "1");
```

Comment:
The machine resolver is not editable. 
Please remove the line
$scope.params.EDITABLE